fe/src/main/java/org/apache/doris/common/publish/FixedTimePublisher.java

```diff
@@ -28,7 +28,7 @@
 public class FixedTimePublisher {
```

Comment:
Looks like this class is not used now.
Maybe we can mark it as @Deprecated now, or even remove it?

Reply:
ok, I will remove it.